### DISCUSSION TITLE
umi4使用antd@5.3.2报错

### DISCUSSION THREAD

**Discussion by weeptao** [umijs/umi#10838] [Q&A] [answered]
使用umi，不是max
按照官网上的，安装其他版本 "antd": "^5.3.2",
在从config导入  `plugins: ['@umijs/plugins/dist/antd',], `
配置开启 `antd：{}`

在使用其他组件时没有报错，使用 `Search`,`Dropdown`时都会报错`Uncaught TypeError: Cannot read properties of undefined (reading 'format')` ，我寻思`format`这个不是格式化时间的吗？我也用到格式化时间啊
引入都是按照官网一模一样的方式引入的
![image](https://user-images.githubusercontent.com/103228833/227416746-58dbfa20-97de-4d1c-b7ff-43615191a8a9.png)

**Comment by fz6m** [comment, marked as answer]
给一个最小复现看看

**Reply by weeptao** [reply]
感觉根本就使用成功，使用umi4默认的antd4版本可以，安装5版本后所有组件都无法使用，都会报这个错误。我就`npm i antd@5.3.2` ，然后就使用了，其他都没变。使用其他版本是需要配置什么吗？
![image](https://user-images.githubusercontent.com/103228833/227430725-cb5defcb-a282-43cc-897c-796e8cd2e1de.png)
![image](https://user-images.githubusercontent.com/103228833/227431348-f3144473-bbd2-4e19-b70f-b71343876fd6.png)

报错：
![image](https://user-images.githubusercontent.com/103228833/227431408-12ac107a-ac4e-4fbe-9f00-e207c0b8fbe6.png)
